src/scripts/components/main/category-navigation/category-navigation.tsx

```diff
@@ -0,0 +1,289 @@
+                styles.category,
+                index === normalizedIndex && showCategories && styles.active,
+              )}
+            >
+              <FormattedMessage id={`tags.${category}`} />
+              <span>
+                <FormattedMessage
+                  id="entries"
+                  values={{
+                    count: entries,
+                  }}
+                />
+              </span>
+            </li>
+          );
+        })}
+      </nav>
+
+      <div
+        className={cx(
+          styles["category-navigation"],
+          // Apply different classes based on the navigation state
+          // And whether the user has come the content- or category-navigation
+          navigationState === "none" && styles["reveal-from-left"],
+          navigationState === "none" && !showCategories && styles.concealed,
+          navigationState === "back" && styles["reveal-from-right"],
+          navigationState === "forward" && styles.conceal,
+        )}
+        aria-label="Circle Navigation"
+        onTouchStart={handleTouchStart}
+        onTouchMove={handleTouchMove}
+        onTouchEnd={handleTouchEnd}
+        // onWheel={handleScroll}
+        style={
+          {
+            //height: `${_size / 2}px`,
+          }
+        }
+      >
+        <svg
+          className={styles["circle-container"]}
+          id="circle-container"
+          data-current-rotation={rotationOffset}
+          width={_size}
+          height={_size}
+          viewBox={`0 0 ${_size} ${_size}`}
+          style={{
+            //translate: " 0 -50%",
+            //transition: "all 0.5s ease-out",
+            transform: `rotate(${rotationOffset}deg)`,
+          }}
+        >
+          {/* Each category is an "arc", their share of space is proportional to the number of content they have
+          We use SVG to generate the arcs
+          */}
+          {scaledArcs.map((arcAngle, index) => {
+            const endAngle = currentAngle + arcAngle;
+
+            // Convert to radians
+            const startRad = (currentAngle * Math.PI) / 180;
+            const endRad = (endAngle * Math.PI) / 180;
+
+            // Calculate arc points
+            const x1 = _center + _radius * Math.cos(startRad);
+            const y1 = _center + _radius * Math.sin(startRad);
+            const x2 = _center + _radius * Math.cos(endRad);
+            const y2 = _center + _radius * Math.sin(endRad);
+
+            // Create arc path
+            const largeArcFlag = arcAngle > 180 ? 1 : 0;
+            const pathData = `
+              M ${x1} ${y1}
+              A ${_radius} ${_radius} 0 ${largeArcFlag} 1 ${x2} ${y2}
+            `;
+
+            // Update start angle for next arc
+            currentAngle = endAngle + SPACING;
+
+            const isCurrentlySelected = index === normalizedIndex;
+
+            const selectedColor = "rgba(0, 179, 152, 1)";
+            const defaultColor = "rgba(0, 51, 73, 1)";
```

Comment:
Can you move these colors to a constants file or to `variables.css` and apply them using a css class?